test/functional/core/job_spec.lua

```diff
@@ -274,11 +271,8 @@ describe('jobs', function()
   end)
 
   it('can omit options', function()
-    if helpers.pending_win32(pending) then return end
     neq(0, nvim('eval', 'delete(".Xtestjob")'))
-    nvim('command', "call jobstart(['touch', '.Xtestjob'])")
-    nvim('command', "sleep 100m")
-    eq(0, nvim('eval', 'delete(".Xtestjob")'))
+    eq(1, eval("jobstart(['nvim', '-u', 'NONE', '-c', 'q', '.Xtestjob']) > 0"))
```

Comment:
eval returned 0 on the sanitizer build on Travis but eval returned 1 in Appveyor. How is that possible?